File: Data Structures & Algorithms/products-of-array-discluding-self/submission-1.java
class Solution {
    public int[] productExceptSelf(int[] nums) {
        int[] yo = new int[nums.length];
        int l = 1;
        int r= 1;
        for (int i=0; i<nums.length; i++) {
            yo[i]=l;
            l*=nums[i];
        }
        for (int j=nums.length-1; j>=0; j--){
            yo[j]*=r;
            r*=nums[j];
        }
        return yo;
        
    }
}  

File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        l = {}
        y =[]
        for i in range(len(nums)):
            if (target-nums[i] in l):
                return [l.get(target-nums[i]),i]
            l[nums[i]]=i
        
        